### PR TITLE
Fix deploy

### PR DESCRIPTION
**Pull request by asvetlov**
Fix tests running on built wheels, don't upload non-aiohttp wheels

### CODE REVIEW

File: tools/run_docker.sh [hunk 1]
```diff
@@ -17,7 +19,7 @@ do
 done
 
 echo Creating dist folder with privileges of host-machine user
-mkdir dist  # This is required to be created with host-machine user privileges
+mkdir -p dist  # This is required to be created with host-machine user privileges
```
Comment (webknjaz) [inline tools/run_docker.sh:22]: This is not needed. `--path` argument only takes effect if you create subpaths, like `dist/subfolder` and `dist` doesn't exist. This won't change anything for a single atom.

File: tools/run_docker.sh [hunk 2]
```diff
@@ -33,3 +35,5 @@ do
 
     dock_ext_args=""  # Reset docker args, just in case
 done
+
+find dist -type f -not -name "*aiohttp*" | xargs rm -f
```
Comment (webknjaz) [inline tools/run_docker.sh:39]: I've covered this with https://github.com/aio-libs/aiohttp/pull/2334/files#diff-efcf4fcbe5802c1378b182a49aaf85e8R40 (tested with other repo, but should also work)

File: tools/build-wheels.sh [hunk 3]
```diff
@@ -1,4 +1,6 @@
 #!/bin/bash
+set -e
```
Comment (webknjaz) [inline tools/build-wheels.sh:2]: https://coderwall.com/p/fkfaqq/safer-bash-scripts-with-set-euxo-pipefail

Comment (webknjaz) [reply tools/build-wheels.sh:2]: `set -euxo pipefail`

Comment (webknjaz) [reply tools/build-wheels.sh:2]: Even better:
```bash
if [ -n "$DEBUG" ]
then
  set -x
fi

set -euo pipefail
```

File: tools/run_docker.sh (hunk 1 above)
Comment (asvetlov) [reply tools/run_docker.sh:22]: I've added it for running `./tools/run_docker.sh "aiohttp"` locally.
On travis the change makes no harm but I've been tired to run `rm -rf dist` before `run_docker.sh` every time.

File: tools/run_docker.sh (hunk 2 above)
Comment (asvetlov) [reply tools/run_docker.sh:39]: Maybe you mean another link -- but let's keep the line as is for sake of quick merge.
You can polish the line later.
I'm striving to release aiohttp 2.3 ASAP

File: tools/build-wheels.sh (hunk 3 above)
Comment (asvetlov) [reply tools/build-wheels.sh:2]: Do you mean I should change it to `set -euxo pipefail`?
Sounds good but please left it for next PRs.
I hope you'll add the proposal into your work on travis polishing.
Right now I want to fix travis deployment for getting 2.3 release ASAP.
Every change requires a waiting for travis for completion.

File: tools/run_docker.sh (hunk 2 above)
Comment (kxepal) [reply tools/run_docker.sh:39]: find has `-delete` option, no need to pipe to xargs.

Comment (webknjaz) [reply tools/run_docker.sh:39]: yes, but the match expression is a bit more complete :)

File: tools/build-wheels.sh (hunk 3 above)
Comment (webknjaz) [reply tools/build-wheels.sh:2]: sure

File: tools/run_docker.sh (hunk 1 above)
Comment (webknjaz) [reply tools/run_docker.sh:22]: Yea, I see. But I'm talking about the `-p` argument :)

File: tools/run_docker.sh (hunk 2 above)
Comment (webknjaz) [reply tools/run_docker.sh:39]: `find /io/dist -maxdepth 1 -type f ! -path '/io/dist/'"$package_name"'-*-manylinux1_${arch}.whl' -exec rm -fv {} \;` @asvetlov I'm referring to this line

File: tools/run_docker.sh (hunk 1 above)
Comment (asvetlov) [reply tools/run_docker.sh:22]: I'm also talking about `-p`.
With it `mkdir` doesn't create `dist` folder if it already exists.

**Pull request merged**